src/router.js

```diff
@@ -45,6 +46,7 @@ const Routes = () => (
       <Route path="/forgot-password" component={PasswordReset} />
       <Route path="/register" component={RegisterPage} />
       <Route path="/new-password" component={CreateNewPassword} />
+      <ProtectedRoute isAllowed={hasToken} path="/projects" component={AdminProjectsPage} />
```

Comment:
This line also get s affected;
```suggestion
      <ProtectedRoute isAllowed={hasToken} path="/clusters/:clusterID/projects" component={AdminProjectsPage} />
```